Sources/Config.swift

```diff
@@ -0,0 +1,194 @@
+//
```

Comment:
Might be worth splitting this file up right now, as it could get quite long once we add more features such as pipe-ing, etc..?

Reply:
Yeah, good idea, will try